ee/modules/600-flant-integration/templates/madison-proxy-grafana-alert-channel.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and (.Values.flantIntegration.madisonAuthKey) (and ($.Values.global.enabledModules | has "prometheus-crd") ($.Values.global.enabledModules | has "prometheus")) }}
+apiVersion: deckhouse.io/v1alpha1
+kind: GrafanaAlertsChannel
+metadata:
+  name: madison-proxy
+{{ include "helm_lib_module_labels" (list . (dict "app" "madison-proxy")) | indent 2 }}
+spec:
+  description: "Channel to send alerts to Polk. Auto-created by flant-integration module"
+  type: PrometheusAlertManager
+  # we do not need receive [OK] alerts in madison (and do not want show in polk)
```

Comment:
I did not get the point. Why don't we need `ok`s?

Reply:
In Polk, an incident is considered complete when no more events are received by it, therefore the OK alert has no semantic meaning in the shelf. In my mind.

This will be an additional alert that the person on duty will have to pay attention to and simply close it in 100% of cases.

Reply:
I was wrong. Thank you, Maksim.
For closing event in alert manager (madison is alert manager) we need to send `OK` message
 
I've reverted [disableResolveMessage](https://github.com/deckhouse/deckhouse/pull/402/files#diff-16cedd54eda1ebda5decdedddb4ff4145e08242b5ce8c3fa6bf489ddff38ca14R42) default to `false` (always send `OK's`)
and I've removed this flag for madison-proxy manifest. And madison-proxy will send OK messages.

Reply:
@nabokihms ^ fyi